src/java/org/apache/cassandra/db/ColumnFamilyStore.java

```diff
@@ -817,14 +829,6 @@ public static void  scrubDataDirectories(TableMetadata metadata) throws StartupE
                     if (!file.tryDelete())
                         logger.warn("could not delete {}", file.absolutePath());
         }
-
```

Comment:
why is this being removed?

Reply:
can return back

Reply:
ah right I see what you ask, like why I removed that block of the code? Because it is not necessary as I moved the logic up into scrubDataDirectories where I am fitlering indexes and we will scrub it with all tables as well.